src/cudamatrix/cublas-wrappers.h

```diff
@@ -278,25 +278,14 @@ inline cusparseStatus_t cusparse_csrmm2(cusparseHandle_t handle,
   if (status != CUSPARSE_STATUS_SUCCESS) return status;
 
   size_t buffer_size;
```

Comment:
This seeems to be just removing the original support for ALG2... and you said you were compiling with 10.2; in this case surely CUDA_VERSION should not be 11000?  Maybe you changed the installation in some default place but failed to reconfigure?